<?xml version="1.0" encoding="UTF-8" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>[PI] Compile error</title>
    <link>https://forum.moparisthebest.com/t/pi-compile-error/412229</link>
    <description>What can cause it?
[code][27.08.11 8:58]: Initializing start up...
[27.08.11 8:58]: Loading scripts...
.\Data\scripts\player\eat\eat.pyscript:
.\Data\scripts\player\items\spade.pypt:
.\Data\scripts\player\npc\npc1.pycript:
.\Data\scripts\player\npc\npc2.pycript:
.\Data\scripts\player\npc\npc3.pycript:
.\Data\scripts\player\objects\objectclick1.py
.\Data\scripts\player\objects\objectclick2.py
.\Data\scripts\player\objects\objectclick3.py
.\Data\scripts\player\skills\herblore.py
.\Data\scripts\player\skills\mining.py:
[27.08.11 8:58]: Exception in thread &quot;main&quot; Traceback (innermost last):
  File &quot;&lt;iostream&gt;&quot;, line 4, in ?
ImportError: no module named random
Press any key to continue . . .[/code]</description>
    
    <lastBuildDate>Sat, 27 Aug 2011 06:01:43 +0000</lastBuildDate>
    <category>Runescape</category>
    <atom:link href="https://forum.moparisthebest.com/t/pi-compile-error/412229.rss" rel="self" type="application/rss+xml" />
      <item>
        <title>[PI] Compile error</title>
        <dc:creator><![CDATA[@Rad_X Rad X]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/rad_x">@Rad_X</a> wrote:</p>
          <blockquote>
              <p>What can cause it?</p>
<p><code>[27.08.11 8:58]: Initializing start up...
[27.08.11 8:58]: Loading scripts...
.\Data\scripts\player\eat\eat.pyscript:
.\Data\scripts\player\items\spade.pypt:
.\Data\scripts\player\npc\npc1.pycript:
.\Data\scripts\player\npc\npc2.pycript:
.\Data\scripts\player\npc\npc3.pycript:
.\Data\scripts\player\objects\objectclick1.py
.\Data\scripts\player\objects\objectclick2.py
.\Data\scripts\player\objects\objectclick3.py
.\Data\scripts\player\skills\herblore.py
.\Data\scripts\player\skills\mining.py:
[27.08.11 8:58]: Exception in thread "main" Traceback (innermost last):
  File "&lt;iostream&gt;", line 4, in ?
ImportError: no module named random
Press any key to continue . . .</code></p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/pi-compile-error/412229/1">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/pi-compile-error/412229/1</link>
        <pubDate>Sat, 27 Aug 2011 06:01:43 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-412229-1</guid>
        <source url="https://forum.moparisthebest.com/t/pi-compile-error/412229.rss">[PI] Compile error</source>
      </item>
  </channel>
</rss>
